src/01_bignum.py

```diff
@@ -1,4 +1,5 @@
 # Print out 2 to the 65536 power
 # (try doing the same thing in the JS console and see what it outputs)
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+print(2**65536)
```

Comment:
That's a big number! Good job! 
![Screen Shot 2019-12-09 at 10 01 45 AM](https://user-images.githubusercontent.com/16110931/70446353-f6effd00-1a6a-11ea-8e22-96abcda88fcd.png)